mods/d2k/fluent/chrome.ftl

```diff
@@ -63,99 +63,111 @@ label-mute-indicator = Audio Muted
-   .tooltipdesc = Selected units will move to the desired location
+    .tooltip = Attack Move
+    .tooltipdesc =
+    Selected units will move to the target location,
     and attack any enemies they encounter en route.
 
-    Hold <(Ctrl)> while targeting to order an Assault Move
-    that attacks any units or structures encountered en route.
+    Left-click the icon, then right-click on the target.
 
-    Left-click icon then right-click on target location.
+    Hold <(Ctrl)> while targeting to issue an Assault Move,
+    to attack any units or structures encountered en route.
 
 button-command-bar-force-move =
-   .tooltip = Force Move
-   .tooltipdesc = Selected units will move to the desired location
-     - Default activity for the target is suppressed
-     - Vehicles will attempt to crush enemies at the target location
-     - Deployed thumpers will undeploy and move to the target location
+    .tooltip = Force Move
+    .tooltipdesc =
+    Selected units will move to the target location.
+    - Vehicles attempt to crush enemies at the target location.
+    - Deployed thumpers undeploy and move to the target location.
 
-    Left-click icon then right-click on target.
-    Hold <(Alt)> to activate temporarily while commanding units.
+    Left-click the icon, then right-click on the target.
+    Hold <(Alt)> to temporarily enable this command while controlling units.
 
 button-command-bar-force-attack =
-   .tooltip = Force Attack
-   .tooltipdesc = Selected units will attack the targeted unit or location
-     - Default activity for the target is suppressed
-     - Allows targeting of own or ally forces
+    .tooltip = Force Attack
+    .tooltipdesc =
+    Selected units will attack the target unit or location.
+    - Allows targeting of allied or own forces.
+    - Long-range artillery will fire at the target
+     location, ignoring units and structures.
 
-    Left-click icon then right-click on target.
-    Hold <(Ctrl)> to activate temporarily while commanding units.
+    Left-click the icon, then right-click on the target.
+    Hold <(Ctrl)> to temporarily enable this command while controlling units.
 
 button-command-bar-guard =
-   .tooltip = Guard
-   .tooltipdesc = Selected units will follow the targeted unit.
+    .tooltip = Guard
+    .tooltipdesc =
+    Selected units will follow and protect the target unit.
 
-    Left-click icon then right-click on target unit.
+    Left-click the icon, then right-click on the target unit.
 
 button-command-bar-deploy =
-   .tooltip = Deploy
-   .tooltipdesc = Selected units will perform their default deploy activity
-     - MCVs will unpack into a Construction Yard
-     - Thumpers will start or stop attracting worms
-     - Devastators will become immobilized and explode
+    .tooltip = Deploy
+    .tooltipdesc =
+    Selected units will perform their default deploy action:
+    - MCVs unpack into Construction Yards.
+    - Construction Yards repack into MCVs.
```

Comment:
The existing tooltip was correct: this feature doesn't exist in D2k.

Reply:
hmm, that's weird